src/db/models/draftarticles.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const Draftarticles = sequelize.define('Draftarticles', {
+    id: {
+      type: DataTypes.UUID,
+      defaultValue: DataTypes.UUIDV4,
+      allowNull: false,
+      primaryKey: true
+    },
+    title: {
+      type: DataTypes.STRING,
+      allowNull: false
+    },
+    body: {
+      type: DataTypes.TEXT,
+      allowNull: false
+    },
+    author: {
+      type: DataTypes.UUID,
+      allowNull: false
+    },
+    slug: {
+      type: DataTypes.STRING,
+      allowNull: false
+    }
+  }, {});
+  Draftarticles.associate = function(models) {
+    Draftarticles.hasMany(models.Rating, {
+      foreignKey: 'post'
```

Comment:
use a more descriptive variable not 'post' for anything attached to the post variable